Cargo.toml

```diff
@@ -1,7 +1,39 @@
+[package]
+name = "runwasi"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+containerd-shim = "0.3.0"
+containerd-shim-wasm = { path = "crates/containerd-shim-wasm" }
+log = "0.4"
+ttrpc = "0.6"
+wasmtime = "^0.35"
+wasmtime-wasi = "^0.35"
+wasi-common = "^0.35"
+chrono = "0.4.19"
+anyhow = "1.0"
+
+[dev-dependencies]
+tempfile = "3.0"
+
 [workspace]
 members = [
-    "crates/containerd-shim-wasmtime-v1",
+    "crates/containerd-shim-wasm",
 ]
 
+[[bin]]
```

Comment:
@Mossaka ~ Why not use workspaces for this, so we can separate dependencies?

Reply:
Thanks for your idea! I would like this shim implementation to be in the same workspace because they share the same `WasiInstance` config, and any change to the dependency should be reflect to all three binaries. When we migrate the Spin Shim and the ASP.NET shim to this repo, we will make each individual shim a workspace, with separate dependencies. Does this make sense?